lasio/las.py

```diff
@@ -210,6 +223,13 @@ def read(
                     logger.debug("Storing reference and returning later...")
                     data_section_indices.append(i)
 
+                # Initial stub for parsing las30 data. This is probably a
+                # transitional section that will merge with 1.2/2.0 data
+                # parsing once fully functional
+                elif section_type == "Las30_Data":
+                    logger.debug("Storing Las30_Data reference and returning later...")
```

Comment:
refer to other comment - small point but can we make it `3` instead of `30`?

Reply:
Made the change in 97a72d9.